hackage-repo-tool/src/Hackage/Security/RepoTool/Util/IO.hs

```diff
@@ -62,10 +69,16 @@ createSymbolicLink :: (FsRoot root, FsRoot root')
                    -> Path root' -- ^ Link location
                    -> IO ()
 createSymbolicLink linkTarget linkLoc = do
+#ifndef mingw32_HOST_OS
     createDirectoryIfMissing True (takeDirectory linkLoc)
     linkTarget' <- toAbsoluteFilePath linkTarget
     linkLoc'    <- toAbsoluteFilePath linkLoc
     Posix.createSymbolicLink linkTarget' linkLoc'
+#else
+    error $ "Cannot create symbolic links on Windows"
```

Comment:
@23Skidoo this makes  `symlink-cabal-local-repo` command fail. Not that it's useful on Windows, but I'd much happier disabling that codepath cleanly (i.e. disabling the command on windows), and not creating technical debt.

Reply:
Fine, but this `#ifdef` will still be needed.

Reply:
by cleanly i mean: #ifdef the whole function, and see what breaks -> disable that with good error message.

Reply:
OK, I can look into it tomorrow.